### PR TITLE
Update pdfjs-dist to 4.4.168

### PR DESCRIPTION
**Pull request by NicolaiHorlacher**
Updated pdf.js to the newest available version.
The pdf.js css changed, so I updated the values in `vite.config.ts`.
I also removed CSS lines that reference svg images, because they are not needed (?) and cause warings in a vite build. As mentioned in this issue #222

Removed CSS lines:
```css
--editorInk-editing-cursor:url(images/cursor-editorInk.svg) 0 16, pointer;
--editorHighlight-editing-cursor:url(images/cursor-editorTextHighlight.svg) 24 24, text;
--editorFreeHighlight-editing-cursor:url(images/cursor-editorFreeHighlight.svg) 1 18, pointer;

.textLayer.highlighting{
  cursor:var(--editorFreeHighlight-editing-cursor);
}

.textLayer.highlighting:not(.free) span{
    cursor:var(--editorHighlight-editing-cursor);
}

.textLayer.highlighting.free span{
    cursor:var(--editorFreeHighlight-editing-cursor);
}

```

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -30,14 +30,15 @@ export default defineConfig({
...
-          [1362, 1373],
         ],
         annotationLayer: [
-          [1323, 1348],
-          [2543, 2553],
+          [1323, 1343],
+          [1347, 1348],
+          [2549, 2559],
           [350, 712],
         ],
```
Comment (hrynko) [inline vite.config.ts:43]: I think it should be like this (please correct me if I'm wrong):

```suggestion
        textLayer: [
          [1323, 1343],
          [1347, 1348],
          [2546, 2555],
          [231, 349],
        ],
        annotationLayer: [
          [1323, 1343],
          [1347, 1348],
          [2546, 2555],
          [350, 712],
        ],
```

File: package-lock.json
Comment (hrynko) [inline package-lock.json:1]: Please revert and apply [this patch](https://github.com/user-attachments/files/16286902/package-lock.json.patch)

**Review by hrynko — CHANGES_REQUESTED** [review]

File: vite.config.ts (hunk 1 above)
Comment (NicolaiHorlacher) [reply vite.config.ts]: You are right. I was three lines off the mark.
I noticed, that all remaining css variables in the `:root` selector weren't used anywhere in the css files. So I excluded them also.

Removed lines:
```css
:root {
  --outline-width: 2px;
  --outline-color: #0060df;
  --outline-around-width: 1px;
  --outline-around-color: #f0f0f4;
  --hover-outline-around-color: var(--outline-around-color);
  --focus-outline: solid var(--outline-width) var(--outline-color);
  --unfocus-outline: solid var(--outline-width) transparent;
  --focus-outline-around: solid var(--outline-around-width) var(--outline-around-color);
  --hover-outline-color: #8f8f9d;
  --hover-outline: solid var(--outline-width) var(--hover-outline-color);
  --hover-outline-around: solid var(--outline-around-width) var(--hover-outline-around-color);
  --freetext-line-height: 1.35;
  --freetext-padding: 2px;
  --resizer-bg-color: var(--outline-color);
  --resizer-size: 6px;
  --resizer-shift: calc(0px - (var(--outline-width) + var(--resizer-size)) / 2 - var(--outline-around-width))
}
```

That simplifies the css ranges to:
```ts
textLayer: [
  [2546, 2555],
  [231, 349],
],
annotationLayer: [
  [2546, 2555],
  [350, 712],
],
```

File: package-lock.json
Comment (NicolaiHorlacher) [reply package-lock.json:1]: Done.

File: vite.config.ts (hunk 1 above)
Comment (hrynko) [reply vite.config.ts]: Indeed, thanks!

**Review by hrynko — APPROVED** [review]

**Pull request merged**